### PR TITLE
Brain Damage Trait

### PR DESCRIPTION
**Pull request by ghost**
No, I agree. I was having a giggle let's have the real pr. I love the timer idea for traits, this seems like the more active hallucinations trait.

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -11,6 +11,17 @@
...
+/datum/trait/brainproblems
+	name = "Brain Tumor"
+	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some mannitol!"
+	value = -2
+	gain_text = "<span class='danger'>You feel smooth.</span>"
+	lose_text = "<span class='notice'>You feel wrinkled again.</span>"
+	medical_record_text = "Patient has a tumor in their brain that is slowly driving them to brain death."
+
+/datum/trait/brainproblems/on_process()
+	trait_holder.adjustBrainLoss(1)
```
[the remark this answers is not in the document]
Comment (Armhulen) [reply code/datums/traits/negative.dm]: Sure, thanks.

**Comment by Armhulen** [comment]
xhuis heavy sleeper is at the top, so...

**Comment by Xhuis** [comment]
Just making sure! I didn't look in the file to make sure.

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -11,6 +11,17 @@
 	lose_text = "<span class='notice'>You feel awake again.</span>"
 	medical_record_text = "Patient has abnormal sleep study results and is difficult to wake up."
 
+/datum/trait/brainproblems
+	name = "Brain Tumor"
+	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some mannitol!"
+	value = -2
+	gain_text = "<span class='danger'>You feel smooth.</span>"
```
Comment (kevinz000) [inline code/datums/traits/negative.dm]: Okay we get it armhulenn don't need to tell us about yourself too much

**Comment by ExcessiveUseOfCobblestone** [comment]
enjoy your true github ban for using the r word in a PR title.

File: code/datums/traits/negative.dm (hunk 2 above)
Comment (PKPenguin321) [reply code/datums/traits/negative.dm]: Kevin is a smoothbrain

**Comment by optimumtact** [comment]
this isn't the hut you under 9000

**Comment by Armhulen** [comment]
Not getting git banned today

**Comment by AlexTheSysop42** [comment]
yay arm is ungitbanned

**Comment by ExcessiveUseOfCobblestone** [comment]
r* is an alternative to a* if you were unaware

**Comment by Armhulen** [comment]
Not getting git banned today
--




LOL I DIDN'T TYPE THIOS

**Comment by Armhulen** [comment]
ready, btw.

**Pull request merged**